genmbr.py

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env python
 
+import os
+import sys
+
+assert sys.version_info.major >= 3, 'requires Python 3'
+
+def writeU32LE(out, value):
+	out.write(bytes([(value >> shift) & 0xFF for shift in range(0, 32, 8)]))
+
```

Comment:
The [`struct`](https://docs.python.org/3/library/struct.html) module would be a more elegant way of doing `int` to `bytes` conversions.

Reply:
Neat, didn't know about it. Done.

Reply:
I didn't know about it either when I originally wrote the script.